taca/storage/storage.py

```diff
@@ -239,21 +241,22 @@ def _send_to_swestore(file_path, dest, remove=True):
         if os.stat(run).st_mtime < time.time() - (86400 * days):
             _send_to_swestore(run, CONFIG.get('storage').get('irods').get('irodsHome'))
         else:
-            logger.info("Run {} is not {} days old yet. Not archiving"
-                        .format(run, str(days)))
+            logger.info("Run {} is not {} days old yet. Not archiving".format(run, str(days)))
     else:
         rta_file = os.path.join(run, 'RTAComplete.txt')
-        if os.stat(rta_file).st_mtime < time.time() - (86400 * days):
+        if not os.path.exists(rta_file) and not force:
+            logger.warn(("Run {} doesn't seem to be completed and --force option was "
+                      "not enabled, not archiving the run".format(run)))
```

Comment:
second if is `if force or (os.path.exists(rta_file) and os.stat(rta_file).st_mtime < time.time() - (86400 * days)):`, it is checking if the file exist, so it shouldn't crash.